app/commands/compute_color_score.rb

```diff
@@ -0,0 +1,43 @@
+class ComputeColorScore < PowerTypes::Command.new(:user_id, :other_users_ids, :pr_relations,
+                                                  review_month_limit: nil)
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.